### ISSUE TITLE
`--cache-to` option is not available on v8.2.1

### ISSUE THREAD

**Issue by corrupt952** [CircleCI-Public/aws-ecr-orb#290]
### Orb version

v8.2.1

### What happened

This issue only occurs when the version of the aws-ecr Orb is updated, and the following behavior has been confirmed

- that the problem does not occur at least in v8.1.2.
- v8.2.1 and setup_remote_docker [version combination](https://circleci.com/docs/building-docker-images/#docker-version) does not resolve the issue.

I tried to update from `v8.1.2` to `v8.2.1`.

And now I get the error `error: cache export feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")`.
As you can see in the error message, the default driver does not support `--cache-to`.

What makes this issue different from similar issues is that it explicitly specifies `--cache-to` to store the cache in an external repository without enabling DLC.

like this:

```yaml
executors:
  image-builder:
    docker:
      - image: cimg/base:stable

jobs:
  build_and_push_container_image:
    executor: image-builder
    steps:
      - git-shallow-clone/checkout
      - setup_remote_docker
      - aws-ecr/build-and-push-image:
          registry-id: 'AWS_ACCOUNT_ID' # env_name
          region: '$AWS_DEFAULT_REGION'
          repo: << parameters.repo >>
          tag: '$REVISION'
          dockerfile: Dockerfile
          checkout: false
          create-repo: false
          skip-when-tags-exist: false
          docker-login: true
          extra-build-args: >-
            --cache-from type=registry,ref=corrupt952/hoge-build-cache:${CIRCLE_PROJECT_REPONAME}-${CIRCLE_BRANCH//\//-}-<< parameters.env >>
            --cache-to type=registry,mode=max,ref=corrupt952/hoge-build-cach:${CIRCLE_PROJECT_REPONAME}-${CIRCLE_BRANCH//\//-}-<< parameters.env >>
```

As a workaround, I have confirmed that `docker buildx create --use` before `build-and-push-image` will work.

```yaml
jobs:
  build_and_push_container_image:
    executor: image-builder
    steps:
      - git-shallow-clone/checkout
      - setup_remote_docker
      - run: docker buildx create --use
      - aws-ecr/build-and-push-image:
          registry-id: 'AWS_ACCOUNT_ID' # env_name
          region: '$AWS_DEFAULT_REGION'
          repo: << parameters.repo >>
          tag: '$REVISION'
          dockerfile: Dockerfile
          checkout: false
          create-repo: false
          skip-when-tags-exist: false
          docker-login: true
          extra-build-args: >-
            --cache-from type=registry,ref=corrupt952/hoge-build-cache:${CIRCLE_PROJECT_REPONAME}-${CIRCLE_BRANCH//\//-}-<< parameters.env >>
            --cache-to type=registry,mode=max,ref=corrupt952/hoge-build-cach:${CIRCLE_PROJECT_REPONAME}-${CIRCLE_BRANCH//\//-}-<< parameters.env >>
```

### Expected behavior

Hope you can use `--cache-to` even if not specified.

**Comment by brivu** [comment]
Hey @corrupt952 - This has been fixed with PR #293. Thanks for your contribution!

**Comment by corrupt952** [comment]
Thanks for fixing it!
I look forward to the release with this fix :)
